Ginger/Ginger/MainWindow.xaml.cs

```diff
@@ -526,12 +526,9 @@ private void ALMFieldsConfiguration_Click(object sender, RoutedEventArgs e)
 
         private void ALMDefectsProfiles_Click(object sender, RoutedEventArgs e)
         {
-            if(!ALMIntegration.Instance.AlmConfigurations.UseRest && ALMIntegration.Instance.GetALMType() != ALMIntegration.eALMType.Jira)
-            {
-                Reporter.ToUser(eUserMsgKey.ALMDefectsUserInOtaAPI, "");
-                return;
-            }
-            ALMIntegration.Instance.ALMDefectsProfilesPage();
+            //open defect profile page for each almtype
+            ALMDefectsProfilesPage defectsProfilesPage = new ALMDefectsProfilesPage();
```

Comment:
Why this check is removed? i think it is needed for QC. Because for qC Defect opening is supported only if use Rest API

Reply:
The defect profile page should open for any alm type, the check should be while opening defect not here, please suggest.